### PR TITLE
Fix #1977-Improve visibility of Snackbar action text.

### LINKED ISSUE #1977: Snackbar 'undo' button visibility low
**Actual Behaviour**

'undo' button color is bit darker which decreases its visibility

**Expected Behaviour**

To use lighter shades of red for greater visibility

**Screenshots of the issue**

image

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mayank8318**
Fixes #1977 

Changes: Adjust opacity of the text displayed.

Screenshots for the change: 
image

### CODE REVIEW

**Review by scandindian — APPROVED** [review]

**Review by S2606 — APPROVED** [review]

File: android/app/src/main/res/values/color.xml [hunk 1]
```diff
@@ -22,4 +22,5 @@
     <color name="timestamp">#a0a3a7</color>
     <color name="link">#0a80d1</color>
     <color name="card_title">#ffffff</color>
+    <color name="snackbar_action_text">#dcb71c1c</color>
```
Comment (iamareebjamal) [inline android/app/src/main/res/values/color.xml]: This is hardcoded text and won't change if the developer wants to use theming option of app generator

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/values/color.xml (hunk 1 above)
Comment (mayank8318) [reply android/app/src/main/res/values/color.xml]: @iamareebjamal Sorry about that. I will fetch the accent colour dynamically and adjust opacity.

File: android/app/src/main/java/org/fossasia/openevent/utils/Views.java [hunk 2]
```diff
@@ -194,4 +197,10 @@ public static void setEdgeGlowColor(@NonNull EdgeEffectCompat edgeEffect, @Color
         if (effect != null)
             effect.setColor(color);
     }
+
+    public static int getAccentColor(final Context context) {
+        final TypedValue value = new TypedValue();
+        context.getTheme().resolveAttribute(R.attr.colorAccent, value, true);
+        return value.data;
+    }
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/utils/Views.java]: You can use `ContextCompat.getColor(context, R.color.colorAccent)` instead

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/utils/Views.java (hunk 2 above)
Comment (mayank8318) [reply android/app/src/main/java/org/fossasia/openevent/utils/Views.java]: Ok

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**